libqtile/backend/wayland/window.py

```diff
@@ -745,6 +745,9 @@ def unhide(self) -> None:
         if not self.mapped:
             self.surface.map_event.emit()
 
+    def is_visible(self) -> bool:
+        return self.mapped and not self.minimized
```

Comment:
All minimized windows are not mapped by definition, so `self.mapped` is sufficient in this case. At least that's supposed to be the case so if it isn't then I should fix that.

And as we're in the same class may as well skip the property?

```suggestion
        return self._mapped
```